package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "libp2p-discv5",
+  "version": "0.0.1",
+  "description": "An implementation of Discovery V5 using Libp2p components",
+  "main": "index.js",
+  "files": [
+    "src",
+    "dist"
+  ],
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "lint": "tslint --project tsconfig.json",
+    "release": "aegir release",
+    "build": "aegir build",
+    "test": "mocha -r ts-node/register test/**/*.test.ts",
+    "test:node": "aegir test --target node",
+    "test:browser": "aegir test --target browser --target webworker",
+    "tsc": "tsc"
+  },
+  "pre-push": [
+    "lint"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ChainSafe/libp2p-discv5.git"
+  },
+  "keywords": [
+    "discovery",
+    "discv5",
+    "ethereum",
+    "libp2p",
+    "peer",
+    "p2p"
+  ],
+  "author": "ChainSafe Systems",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/ChainSafe/libp2p-discv5/issues"
+  },
+  "homepage": "https://github.com/ChainSafe/libp2p-discv5#readme",
+  "devDependencies": {
+    "@types/chai": "^4.2.0",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^12.0.10",
+    "aegir": "^19.0.3",
+    "assert": "^2.0.0",
+    "chai": "^4.2.0",
+    "ethereumjs-utils": "^5.2.5",
+    "karma": "^4.3.0",
+    "mocha": "^6.2.0",
+    "nyc": "^14.1.1",
+    "ts-node": "^8.3.0",
+    "tslint": "^5.18.0",
+    "typescript": "^3.5.3"
+  },
+  "dependencies": {
+    "@chainsafe/milagro-crypto-js": "^0.1.3",
+    "bcrypto": "^4.1.0",
+    "crypto": "^1.0.1",
+    "debug": "^4.1.1",
+    "dgram": "^1.0.1",
+    "es6-promisify": "^6.0.2",
```

Comment:
can we not use both es6-promisify and promisify-es6 here as well? :sweat_smile:

Reply:
My bad. I keep misrembering the order of the words es6 and promisify.